packages/components/app/styles/components/form/textarea.scss

```diff
@@ -13,6 +13,7 @@
 .hds-form-textarea {
   width: 100%;
   max-width: 100%;
+  min-height: 36px;
```

Comment:
sooner or later (later, probably) we'll have to convert this to a design token shared amongst components (it's starting to become a "standard" size)